src/views/landing/StartApplicationForm.vue

```diff
@@ -58,11 +58,17 @@
       />
     </div>
 
-    <div class="cta">
-      <button class="btn btn-lg btn-cta btn-block text-uppercase" type="submit">Start Application</button>
-    </div>
-    <div class="privacy text-center mt-3">
-      <router-link to="/policy/privacy">Privacy Policy</router-link>
+          <div class="cta">
+            <button
+              class="btn btn-lg btn-cta btn-block text-uppercase"
+              type="submit"
+            >Enroll</button>
+          </div>
```

Comment:
We had agreed that "apply" was better than "enroll". This reverts that change.

Reply:
Ok - I was changing it from "Start Application" to match homepage (back then Enroll).
Ill do another PR.